node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/NodeSerializer.java

```diff
@@ -187,8 +187,7 @@ private void toSlime(Allocation allocation, Cursor object) {
         object.setLong(currentRestartGenerationKey, allocation.restartGeneration().current());
         object.setBool(removableKey, allocation.isRemovable());
         object.setString(wantedVespaVersionKey, allocation.membership().cluster().vespaVersion().toString());
-        // TODO serialize dockerImageRepo
-        //object.setString(wantedDockerImageRepoKey, allocation.membership().cluster().dockerImageRepo().orElse(""));
+        object.setString(wantedDockerImageRepoKey, allocation.membership().cluster().dockerImageRepo().orElse(""));
```

Comment:
I realize the deserializer handles empty string, but consider just to serializing dockerImageRepo if it's not set.

Reply:
Yeah, fixed